contrib/babelfishpg_common/src/datetimeoffset.c

```diff
@@ -832,3 +834,12 @@ EncodeDatetimeoffsetTimezone(char *str, int tz, int style)
 
 	*tmp = '\0';
 }
+
+
+Datum sysdatetimeoffset(PG_FUNCTION_ARGS)
```

Comment:
Do not do this in common. You can include `#include "../../babelfishpg_common/src/datetimeoffset.h"` in TSQL files. It will create a copy of header in TSQL extension so there is no hard dependency